src/seals/diagnostics/base_env.py

```diff
@@ -0,0 +1,100 @@
+"""Minimal base class for simple environments."""
+
+import gym
+from gym.spaces import Discrete
+from gym.utils import seeding
+import numpy as np
+
+from seals.util import sample_distribution
+
+
+class BaseEnv(gym.Env):
```

Comment:
As I mentioned in our call, we'll likely want to separate out the tabular and non-tabular bits. Don't need to do it in this PR, though, since `RiskyPath` is tabular.